support/my-account-and-profile/scim.md

```diff
@@ -18,44 +18,49 @@ description: "Learn how to enable SCIM user provisioning through Okta for your S
-1. In the "SCIM connector base URL" field:
+1. Click "View Setup Instructions" to setup SSO and get the Callback URL and Audience URI
+![](media/scim/credential_details.png)
+
+1. Select the "Provisioning" tab, and from the "Integration" menu click "Configure API Integration".
+![](media/scim/scim_connection.png)
+
+1. Check "Enable API Integration"
+1. In the "Base URL" field:
     - For US-hosted customers, enter `https://api.sparkpost.com/api/v1/users/scim/v2`
     - For EU-hosted customers, enter `https://api.eu.sparkpost.com/api/v1/users/scim/v2`
     - For Enterprise customers, enter `https://<host>/api/v1/users/scim/v2`
     
         *Enterprise customers: note that you must replace < host > with your SparkPost tenant name. Please see your TAM if you need assistance determining your callback URL.*
     
-1. Enter `email` in the "Unique identifier field for users"
-1. Under "Supported provisioning actions", check the following boxes:
-    - "Import New Users and Profile Updates"
-    - "Push New Users"
-    - "Push Profile Updates"
-1. From the "Authentication Mode" drop-down menu, select "HTTP Header".
-1. In the "HTTP Header" section, paste the token you generated [here](#generate-an-authorization-token) into the "Authorization" field.
+1. In the "API Token" field, paste the SCIM token you generated.
+1. Click "Test API Credentials" to verify settings are correct.
 1. Click "Save".
-1. In the "To App" section of the "Provisioning" tab, enable "Create Users", "Update User Attributes", and "Deactivate Users".
-1. In the "To App" section, delete all attribute mappings except "Given name", "Family name", and "Primary email".
-1. To allow updating user roles:
-    - In the "To App" section, select "Go to Profile Editor".
-    - Select "Add Attribute".
-    - Enter the following values:
-        - Data type: string
-        - Display name: role
-        - Variable name: role
-        - External name: role
-        - External namespace: urn:ietf:params:scim:schemas:core:2.0:User
-        - Description: admin, reporting, templates, or developer
-    - Click "Save Attribute".
-1. In the "To Okta" section, delete all attribute mappings except "First name", "Last name", and "Primary email".
-1. In the "To Okta" section, next to "Imported user is an exact match to Okta user if", select "Email matches".
+![](media/scim/scim_test.png)
+
+1. In the "To App" section of the "Provisioning" tab, enable "Create Users", "Update User Attributes", and "Deactivate Users".  Click "Save".
+![](media/scim/to_app_enable.png)
+
+## Import Existing SparkPost users
+1. Select the "Import" tab and click "Import Now"
+![](media/scim/import_users.png)
+
+1. Verify "Okta User Assignments" match existing users or create new users.
+1. Click "Confirm Assignments"
+1. Verify changes and click "Confirm".
+![](media/scim/verify_import.png)
+
+1. Verify users exist in SparkPost application.
+![](media/scim/sparkpost_users.png)
 
 ## Known Issues / Troubleshooting
 - N/A
```

Comment:
in this section should we also have support email address like in the example given by okta.